core/src/main/java/nl/nn/adapterframework/senders/XsltSender.java

```diff
@@ -95,6 +96,13 @@
 	 */
 	@Override
 	public void configure() throws ConfigurationException {
+		ParameterList parameterList = getParameterList();
```

Comment:
Je kan niet zomaar voor elke Xslt een parameter 'environment' toevoegen, hier in de XsltSender.
Als dit voor de EsbSoapWrapperPipe nodig is, dan moet je het daar doen.

Reply:
Ja daar heb je eigenlijk wel gelijk in. Maar het leek me handig om dit generiek te maken omdat je het vaker nodig kunt hebben. Ik stel voor om een attribuut aan de XsltSender toe te voegen waarmee deze parameter toegevoegd wordt (addEnvironmentParameter, default is dan 'false'). 

Accoord?